cristin-person/src/test/java/no/unit/nva/cristin/person/update/CristinPersonPatchJsonCreatorTest.java

```diff
@@ -0,0 +1,66 @@
+package no.unit.nva.cristin.person.update;
+
+import static no.unit.nva.cristin.model.JsonPropertyNames.ID;
+import static no.unit.nva.cristin.person.model.nva.JsonPropertyNames.FIRST_NAME;
+import static no.unit.nva.cristin.person.model.nva.JsonPropertyNames.LAST_NAME;
+import static no.unit.nva.cristin.person.model.nva.JsonPropertyNames.ORCID;
+import static no.unit.nva.cristin.person.model.nva.JsonPropertyNames.PREFERRED_FIRST_NAME;
+import static no.unit.nva.cristin.person.model.nva.JsonPropertyNames.PREFERRED_LAST_NAME;
+import static no.unit.nva.cristin.person.model.nva.JsonPropertyNames.RESERVED;
+import static no.unit.nva.cristin.person.update.CristinPersonPatchJsonCreator.CRISTIN_FIRST_NAME;
+import static no.unit.nva.cristin.person.update.CristinPersonPatchJsonCreator.CRISTIN_FIRST_NAME_PREFERRED;
+import static no.unit.nva.cristin.person.update.CristinPersonPatchJsonCreator.CRISTIN_SURNAME;
+import static no.unit.nva.cristin.person.update.CristinPersonPatchJsonCreator.CRISTIN_SURNAME_PREFERRED;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+
+public class CristinPersonPatchJsonCreatorTest {
+
+    private static final String VALID_ORCID = "1234-1234-1234-1234";
+    private static final String SOME_NAME = randomString();
+    private static final String SOME_OTHER_NAME = randomString();
+    private static final String SOME_PREFERRED_NAME = randomString();
+
+    @Test
+    void shouldParseInputJsonIntoCristinJsonWithCorrectMappingOfFields() {
```

Comment:
Likely don't need this test if you have the full example in the main test

Reply:
This is a Cristin json version test